File: Tugas/Tugas6.js
let nilai = 90;

switch (true) {
    case (nilai >= 90):
        console.log("Nilai sempurna");
        break;
    case (nilai >= 80):
        console.log("kamu dapat B");
        break;
    case (nilai >= 70):
        console.log("kamu dapat C");
        break;
    if (nilai >= 60) {
        console.log("kamu dapat D");
    }
    default:
        console.log("kamu dapat D");
        break;
}

File: Tugas/Tugas6.cpp
#include <iostream>
using namespace std;

int main() {
    int nilai = 95;
    switch (nilai / 10) {
        case 9:
            cout << "Nilai sempurna" << endl;
            break;
        case 8:
            cout << "kamu dapat B" << endl;
            break;
        case 7:
            cout << "kamu dapat C" << endl;
            break;
        case 6:
            cout << "kamu dapat D" << endl;
            break;
        default:
            cout << "kamu dapat E" << endl; 
    }
    return 0;
}
